### PR TITLE
Update component map to support latest Irving

### PR DESCRIPTION
**Pull request by isGabe**
I ran into an issue where Irving would not run on a fresh install, following the [Quick Start instructions](http://storybook.irvingjs.com/?path=/docs/intro-quick-start--page). See https://alleyinteractive.slack.com/archives/C5W23M1E3/p1605297559445200 for details.

### CODE REVIEW

**Review by danbowles — APPROVED** [review]

File: componentMap.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
-import ComponentMap from '@irvingjs/styled-components';
+import createThemedComponentMap from '@irvingjs/styled-components/componentMap';
+import { componentMap as WordPressComponentMap } from '@irvingjs/wordpress';
+import { IntegrationsManager } from '@irvingjs/integrations';
```
Comment (amyevans) [inline componentMap.js:3]: These 2 packages `@irvingjs/integrations` and `@irvingjs/wordpress` aren't in the `package.json` on this repo

Comment (isGabe) [reply componentMap.js:3]: Oh, good catch! That brings up the question: do we want the wordpress package included here?

Comment (amyevans) [reply componentMap.js:3]: I'm not sure, @jameswburke @joemcgill @ostowe any preferences?

Comment (jameswburke) [reply componentMap.js:3]: I didn't include it originally, and once we have a CLI it'll probably be a flag. But 99% of people looking at this are likely going to want it, and it's a lot easier to remove if you don't need it, vs. figuring out _what_ needs to be added.

tldr; let's add it imo
